### PR TITLE
Added touch support to multi-monitor switching

### LINKED ISSUE #655: Cannot perform touch events on non-focused monitor
**Describe the bug**
GNOME is capable of handling multiple monitors, each with their own touch surface. When PaperWM is active, touch screen events are discarded for all apps on displays other than the currently focused one, and I need to move the mouse to the other display to use the touch screen on it.

**To Reproduce**
1. Connect two touch capable displays
2. Open windows on both displays with PaperWM active
3. Move the mouse to one display
4. Try to touch the other display
5. Nothing happens

**Expected behavior**
The other window reacts properly to touch events

**System information:**

```
Distribution: Arch Linux
GNOME Shell 45.0
Display server: Wayland
PaperWM branch/tag: develop
PaperWM commit: c0ce6ab65bbf72bfc59579eb24a119060d68b581
Enabled extensions:
- drive-menu@gnome-shell-extensions.gcampax.github.com
- trayIconsReloaded@selfmade.pl
- arcmenu@arcmenu.com
- nightthemeswitcher@romainvigier.fr
- paperwm@paperwm.github.com
- dash-to-panel@jderose9.github.com
- quick-settings-tweaks@qwreey
- expandable-notifications@kaan.g.inam.org
- quick-settings-avatar@d-go
- caffeine@patapon.info
- reboottouefi@ubaygd.com
- guillotine@fopdoodle.net
- quicksettings-audio-devices-renamer@marcinjahn.com
- gsconnect@andyholmes.github.io
- vertical-workspaces@G-dH.github.com
- mediacontrols@cliffniff.github.com
```

**Additional context**
Native GNOME without PaperWM lets touch events through to unfocused monitors.

### PR DESCRIPTION
**Pull request by Thesola10**
This fixes #655

### CODE REVIEW

File: stackoverlay.js [hunk 1]
```diff
@@ -316,6 +322,12 @@ export class StackOverlay {
         this.setTarget(null);
     }
 
+    touchTriggerPreview() {
+        if (event.type() === Clutter.EventType.TOUCH_BEGIN)
+            return triggerPreview();
+        return Clutter.EVENT_PROPAGATE;
+    }
+
```
Comment (Thesola10) [inline stackoverlay.js:330]: This can be removed, but the branch protection rules prevent me from doing so

Comment (jtaala) [reply stackoverlay.js:330]: hmm, you couldn't delete these lines and then push a new commit?  Unless you were trying to force-push?

Anyways, I see you've created a new PR.

Comment (Thesola10) [reply stackoverlay.js:330]: I can't even push normally, that's the thing. Once the branch is created, it goes into a protection rule and I can't push anything to it anymore :(

Comment (jtaala) [reply stackoverlay.js:330]: hmm - let me have a look (I might have borked the rules).

Comment (jtaala) [reply stackoverlay.js:330]: Yeah, I think I messed up the rules.  You should be good now :+1:

**Pull request closed without merging**